pom.xml

```diff
@@ -93,6 +93,11 @@
       <artifactId>jsr305</artifactId>
       <version>3.0.1</version>
     </dependency>
+		<dependency>
+		    <groupId>org.json</groupId>
```

Comment:
I don't think so. GSON would be dramatic overkill for reading a single value. We could use JSR 353 if the license (GPL, CDDL) isn't a problem.